merkle/sparse_merkle_tree_test.go

```diff
@@ -251,10 +251,10 @@ func TestInclusionProofGetsIncorrectNode(t *testing.T) {
 		const key = "SomeArbitraryKey"
 		proof, err := r.InclusionProof(rev, []byte(key))
 		if err == nil {
-			t.Fatalf("InclusionProof() should've returned an error due to incorrect node from storage layer: %v", proof)
+			t.Errorf("InclusionProof() = %v, nil. Want: nil, error", proof)
 		}
 		if !strings.Contains(err.Error(), "1 remain(s) unused") {
-			t.Fatalf("Saw unexpected error: %v", err)
+			t.Errorf("InclusionProof() = %v, %v. Want: 1 remain(s) unused", proof, err)
```

Comment:
Any reason not to make the two error messages more similar ("want: nil, '1 remain(s) unused'")?